ci/provisioning/post_provision_config_common_functions.sh

```diff
@@ -193,7 +193,8 @@ set_local_repo() {
     version=${version%%.*}
     if [ "$repo_server" = "artifactory" ] &&
        [ -z "$(rpm_test_version)" ] &&
-       [[ ${CHANGE_TARGET:-$BRANCH_NAME} != weekly-testing* ]]; then
+       { [[ ${CHANGE_TARGET:-$BRANCH_NAME} != weekly-testing* ]] ||
+         [[ ${CHANGE_TARGET:-$BRANCH_NAME} != provider-testing* ]]; }; then
```

Comment:
Hrm.  I think this actually needs to be:

```suggestion
       [[ ${CHANGE_TARGET:-$BRANCH_NAME} != weekly-testing* ]] &&
       [[ ${CHANGE_TARGET:-$BRANCH_NAME} != provider-testing* ]]; then

```